libs/base/fixed.ts

```diff
@@ -60,6 +60,12 @@ namespace Fx {
         else
             return b
     }
+    export function floor(v: Fx8): Fx8 {
+        return ((v as any as number) & ~0xff) as any as Fx8;
+    }
+    export function ceil(v: Fx8): Fx8 {
+        return (v as any as number) & 0xff ? Fx.floor(Fx.add(v, Fx.oneFx8)) : v;
+    }
```

Comment:
@mmoskal do these Fx8 additions look ok?

Reply:
Yes, not sure about negative numbers though